markdown-link-check

```diff
@@ -51,36 +51,40 @@ program
                 parsed.pathname = parsed.pathname.substr(0, parsed.pathname.lastIndexOf('/') + 1);
             }
             opts.baseUrl = url.format(parsed);
-        } catch (err) { /* ignore error */ 
+        } catch (err) { /* ignore error */
```

Comment:
This change is not related but a styling.

Reply:
Absolutely it's part of b7e4256 which atomically indicates changes related to styling. 

Does your comments mean I should issue a PR simply for styling ?

Reply:
No, I mean it's not related to the topic, however I'm not the maintainer/author, just leave my comment without any request ;)